src/ExtraConfigurator.php

```diff
@@ -153,34 +145,31 @@ public static function parseAllEnvironmentParameters()
 	}
 
 	/**
-	 * @return bool
+	 * @return bool|string
 	 */
 	public static function parseEnvDebugMode()
 	{
 		$debug = getenv('NETTE_DEBUG');
-		if ($debug !== FALSE) {
+		if ($debug !== false) {
 			return self::parseDebugValue($debug);
 		}
 
-		return FALSE;
+		return false;
 	}
 
 	/**
-	 * HELPERS ******************************************************************
+	 * @return bool|string
 	 */
-
-	/**
-	 * @param mixed $debug
-	 * @return mixed
-	 */
-	public static function parseDebugValue($debug)
+	public static function parseDebugValue(string $debug)
 	{
 		$value = $debug;
 
 		if (strtolower($value) === 'true' || $value === '1') {
-			$debug = TRUE;
-		} else if (strtolower($value) === 'false' || $value === '0') {
-			$debug = FALSE;
+			$debug = true;
+		} else {
```

Comment:
Wut?

Reply:
Phpstorm did it without warning. I don't understand why, my configuration should only convert `else if` to `elseif`